specification/containerregistry/resource-manager/Microsoft.ContainerRegistry/preview/2019-06-01-preview/examples/RegistriesScheduleRun_Task.json

```diff
@@ -0,0 +1,57 @@
+{
+  "parameters": {
+    "subscriptionId": "4385cf00-2d3a-425a-832f-f4285b1c9dce",
+    "resourceGroupName": "myResourceGroup",
+    "registryName": "myRegistry",
+    "api-version": "2019-06-01-preview",
+    "runRequest": {
+      "type": "TaskRunRequest",
+      "overrideTaskStepProperties": {
+        "file": "overriddenDockerfile",
+        "target": "build",
+        "arguments": [
+          {
+            "name": "mytestargument",
+            "value": "mytestvalue",
+            "isSecret": false
+          },
+          {
+            "name": "mysecrettestargument",
+            "value": "mysecrettestvalue",
+            "isSecret": true
+          }
+        ],
+        "values": [
+          {
+            "name": "mytestname",
+            "value": "mytestvalue",
+            "isSecret": false
+          },
+          {
+            "name": "mysecrettestname",
+            "value": "mysecrettestvalue",
+            "isSecret": true
+          }
+        ]
+      },
+      "updateTriggerToken": "aGVsbG8gd29ybGQ=",
```

Comment:
Model Validator is throwing the right error. `updateTriggerToken` is a property of  Model `OverrideTaskStepProperties`, whereas here it is defined as a property of runRequest. I think this should be defined after Line 35, inside property `overrideTaskStepProperties`